addons/website_sale/models/sale_order.py

```diff
@@ -79,13 +79,15 @@ def _compute_cart_info(self):
             order.cart_quantity = int(sum(order.mapped('website_order_line.product_uom_qty')))
             order.only_services = all(l.product_id.type == 'service' for l in order.website_order_line)
 
-    @api.depends('order_line.price_unit', 'order_line.tax_id', 'order_line.discount', 'order_line.product_uom_qty')
+    @api.depends('order_line.price_total', 'order_line.price_subtotal')
     def _compute_amount_delivery(self):
-        for order in self:
-            if self.env.company.show_line_subtotals_tax_selection == 'tax_excluded':
-                order.amount_delivery = sum(order.order_line.filtered('is_delivery').mapped('price_subtotal'))
+        # FIXME wtf is this crap stored ?
```

Comment:
![do-or-do-not-there-is-no-todo](https://github.com/odoo/odoo/assets/15896813/c6f027e5-e16a-42e6-adba-3adcf905f57a)